liblangutil/ErrorReporter.h

```diff
@@ -124,12 +124,18 @@ class ErrorReporter
 
 	void clear();
 
-	/// @returns true iff there is any error (ignores warnings and infos).
+	/// @returns true if there is any error (ignores warnings and infos).
```

Comment:
It's not like it matters much, but this was probably intentional :)
https://en.wikipedia.org/wiki/iff

Reply:
TIL :)